### PR TITLE
Add Recorder Settings doc

### PR DESCRIPTION
**Pull request by jasonxuclickcorp**

### CODE REVIEW

File: doc/tutorial/recorder/settings.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# Settings
+
+## Version Requirement
+[Clicknium Visual Studio Code Extension](https://marketplace.visualstudio.com/items?itemName=ClickCorp.clicknium) >= 0.1.9
+
+[Clicknium Python Module](https://pypi.org/project/clicknium/) >= 0.1.9
+
+## Overview
+
+Support for setting up a shortcut key for image and UI locator capture.
```
Comment (robinbob) [inline doc/tutorial/recorder/settings.md:14]: The shortcut key for image and UI locator capturing can be customized through this configuration.

File: doc/tutorial/recorder/settings.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- The menu list as below:  
+&emsp;  ![recorder window](../../img/recorder_menu_list.png)
+
+- Click the `Settings` button to open the Settings window.  
+&emsp;  ![recorder window](../../img/recorder_settings_button.png)
+
+- Select your own shortcut key.  
+&emsp;  ![recorder window](../../img/recorder_settings_selection.png)
+
```
Comment (robinbob) [inline doc/tutorial/recorder/settings.md:35]: add a remarks: 

The modifier key/keys can't be same for image and UI locator capturing actions. For e.g., if you choose Ctrl+Click to add UI locator, you can't select Ctrl+Drag Selection for image locator capturing.

File: doc/tutorial/recorder/settings.md (hunk 1 above)
Comment (jasonxuclickcorp) [reply doc/tutorial/recorder/settings.md]: Fixed, thanks.

File: doc/tutorial/recorder/settings.md (hunk 2 above)
Comment (jasonxuclickcorp) [reply doc/tutorial/recorder/settings.md:36]: Fixed, thanks.

**Review by robinbob — APPROVED** [review]

**Pull request merged**